package-sbom/syft-provider.go

```diff
@@ -178,11 +178,11 @@ func GenerateSBOM(config util.Config) ([]byte, error) {
 			} else if scanType == "php" {
 				syftArgs = append(syftArgs, "--catalogers", "php-composer-installed-cataloger", "--catalogers", "php-composer-lock-cataloger")
 			} else if scanType == "golang" {
-				syftArgs = append(syftArgs, "--catalogers", "go-mod-file-cataloger")
+				syftArgs = append(syftArgs, "--catalogers", "go-mod-file-cataloger", "--catalogers", "go-module-binary-cataloger")
```

Comment:
Can you remove `go-module-binary-cataloger`?
We removed this because it takes lots of memory